spyder/api/plugins/new_api.py

```diff
@@ -664,6 +664,15 @@ def get_font(cls, rich_text=False):
 
         return get_font(option=option, font_size_delta=font_size_delta)
 
+    def get_command_line_options(self):
+        """
+        Get command line options passed by the user when they started
+        Spyder in a system terminal.
+
+        See app/cli_options.py for the option names.
+        """
+        return self._main._cli_options
```

Comment:
Since `self._main` could be `None` probably a validation should be done here to check if that's the case and return a dict with all the options in `None` or with their default value?

This could help to avoid doing try-excepts multiple times like at https://github.com/spyder-ide/spyder/pull/17657/files#diff-4558521b03d8be17b33525c99826bbaf4820514f03b80f9f43fba04d9078302a and https://github.com/spyder-ide/spyder/pull/17657/files#diff-bbd9b2f80f39ed23ac76172ae51803f6bfe63cb9cbfd8922278506c3cd681f95

Reply:
Good point, I haven't thought about it. Instead of creating an ad hoc dict (which could become outdated quickly) I could import `get_options` here (from `app/cli_options.py`) to return the default cli options instead.

What do you think?

Reply:
Sounds good :+1: